includes/admin/class-wc-rest-upe-flag-toggle-controller.php

```diff
@@ -91,6 +91,10 @@ public function set_flag( WP_REST_Request $request ) {
 
 		update_option( 'woocommerce_stripe_settings', $settings );
 
+		if ( class_exists( 'WC_Stripe_UPE_Availability_Note' ) ) {
```

Comment:
```suggestion
        // adding `class_exists` because the class might not be present in older versions of WC
		if ( class_exists( 'WC_Stripe_UPE_Availability_Note' ) ) {
```